.github/workflows/build.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         with:
           name: test-failures
           path: |
-            **/build/reports/tests/test/
+            **/build/reports/tests/*/
             **/out/failures/
```

Comment:
We'll need the reports from individual tests too, they're Gradle projects with Android tests ezecuted, does this ** include those?